### PR TITLE
Test/main component

### PR DESCRIPTION
**Pull request by gabriofr**
To do: 
* [x] Add tests for H,M,L filters
* [x] Create dataBuilders.
* [-] Add test for year filter.
* [-] Add test for treatment units filter.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -268,6 +268,7 @@ function APP() {
     update_selection_bar_height(top);
   }, [selection_bar_dim]);
 
+  console.log("This:", { agg_data, description });
```
Comment (yte0) [inline src/App.tsx:271]: rm console.log?

**Review by yte0 — APPROVED** [review]
looks great to me

File: src/App.tsx (hunk 1 above)
Comment (gabriofr) [reply src/App.tsx]: wupsi, har fjerna den nå :)

**Comment by gabriofr** [comment]
@yte0 Fixed the problem with Node10 builds, the .flat() array function is not supported in es5 so I replaced it with a reducer function. 
Should be good to go now, have a look in and Squash and merge as you see fit :)

**Pull request merged**